pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -427,62 +443,16 @@ func (n *NetworkPolicyController) GetConnectedAgentNum() int {
 	return n.appliedToGroupStore.GetWatchersNum()
 }
 
-// toGroupSelector converts the podSelector, namespaceSelector and externalEntitySelector
```

Comment:
If possible, better to have a seperate PR to reduce code redundancy via a seperate PR so this PR can be cleaner.

Reply:
Done.